src/test/java/edu/cornell/kfs/module/purap/jaggaer/supplier/xml/SupplierSyncMessageTest.java

```diff
@@ -338,16 +338,16 @@ private TaxInformationList buildTaxInformationList() {
         TaxDocument document = new TaxDocument();
         document.setIsChanged(T_TRUE);
 
-        document.setAttachments(buildAttachments("super cool tax document"));
+        document.setAttachmentList(buildAttachments("super cool tax document"));
 
         info.setTaxDocument(document);
 
-        taxList.getTaxInformation().add(info);
+        taxList.getTaxInformations().add(info);
         return taxList;
     }
 
-    private Attachments buildAttachments(String attachmentName) {
-        Attachments attachments = new Attachments();
+    private AttachmentList buildAttachments(String attachmentName) {
+        AttachmentList attachments = new AttachmentList();
```

Comment:
Should this variable be renamed to "attachmentList" for consistency?

Reply:
ok